<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWidgeClient</class>
 <widget class="QWidget" name="ChatWidgeClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="listMessages"/>
     </item>
     <item>
      <widget class="QWidget" name="wdgSend" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,3,1">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Message</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lnMessage">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSend">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
